### ISSUE TITLE
Debugging in PyCharm question (breakpoints)

### ISSUE THREAD

**Issue by gmmueller** [koterpillar/aloe#141]
Is it possible to debug in PyCharm via breakpoints?  I did some googling and only really found where people could debug utilizing PyCharm Pro, but that only supports Lettuce and Behave.  Basically, I am looking for the ability to put a breakpoint within a step definition and step through the code.

Thanks

**Comment by koterpillar** [comment]
You can debug using pdb with setting breakpoints, stepping through and the like working absolutely the same way as with the Python files.

I'm not familiar with PyCharm, so perhaps you can give it a try and report back? At least stepping through should work properly.

**Comment by gmmueller** [comment]
Yes, I understand that pdb could be used, but it isn't ideal in my particular situation.  PyCharm allows you to set breakpoints in the IDE similar to Visual Studio, it is more user friendly than using pdb which is important in my situation.  Is it possible to run aloe via python instead of using the executable (like calling Runner). I have not had a chance to try anything myself, so thank you for taking the time to respond.

**Comment by sgpy** [comment]
@gmmueller Not sure if you figured it out :)?
